web/frontend/styles/casebook.scss

```diff
@@ -143,7 +143,7 @@ main > header.casebook {
       color: $light-blue;
     }
     &:focus {
-      @include generic-focus-styles;
+      // @include generic-focus-styles;
```

Comment:
Get rid of the focus-ring around the action buttons (this shows up longer for exports).

Reply:
sorry to butt in: will keyboard users still be able to tell they are focused on the button?

Reply:
<img width="163" alt="image" src="https://user-images.githubusercontent.com/214293/153064384-d790b668-4776-48a4-8841-b7593a88a17c.png">
Cancel is focused. This may be too subtle.

Reply:
🤷‍♀️ 👍 at least there is still something.

Reply:
I rolled it back. It still looks just fine.